starter-code/index.html

```diff
@@ -40,14 +40,14 @@ <h1>Pizza Builder</h1>
       <h2>Your pizza's price</h2>
 
       <b>$10 cheese pizza</b>
-      <ul>
-        <li>$1 pepperonni</li>
-        <li>$1 mushrooms</li>
-        <li>$1 green peppers</li>
-        <li>$3 white sauce</li>
-        <li>$5 gluten-free crust</li>
+      <ul class="ul">
+        <li id="li-1">$1 pepperonni</li>
+        <li id="li-2">$1 mushrooms</li>
+        <li id="li-3">$1 green peppers</li>
+        <li id="li-4">$3 white sauce</li>
+        <li id="li-5">$5 gluten-free crust</li>
       </ul>
```

Comment:
I think naming these `li-1` is a wee bit redundant, you are able to select them using the right query methods (e.g. `eq(i)` which gets the element at index `i`.  If anything, I would give them a class that corresponds to the ingredient or to the ingredient button, if you find that useful in your implementation.